docs/migrating_2.rst

```diff
@@ -35,6 +35,8 @@ Overview
 - ``ChannelType.store`` is removed.
 - ``AppInfo.summary``, ``AppInfo.guild_id``, ``AppInfo.primary_sku_id`` and ``AppInfo.slug`` are removed.
 - ``PartialAppInfo.summary`` is removed.
+- :meth:`Messageable.pins` has been extracted out from :class:`Messageable` and moved to :class:`PinsMixin`
```

Comment:
```suggestion
- :meth:`abc.Messageable.pins` has been extracted out from :class:`abc.Messageable` and moved to :class:`PinsMixin`
```
as it cannot resolve

also consider adding a longer explanation for this in the `Changes` header below